pyramid_heroku/migrate.py

```diff
@@ -15,9 +16,10 @@ class Heroku(object):
     api_endpoint = "https://api.heroku.com"
 
     def __init__(self, app_name, ini_file, app_section):
-        # type: (str, str)-> ()
+        # type: (str, str, str)-> None
```

Comment:
Please use Python-3 style type hinting that is inline, not in comments.

Reply:
I just went with the existing style.
Should I make similar changes to rest of the file as well ?

Reply:
Yes.